Sources/CGFloatExtensions.swift

```diff
@@ -62,4 +62,22 @@ extension CGFloat {
     public static func random(within: ClosedRange<CGFloat>) -> CGFloat {
         return CGFloat.random() * (within.upperBound - within.lowerBound) + within.lowerBound
     }
+    
+    /**
+      EZSE :Returns the shortest angle between two angles. The result is always between
+      -π and π.
+     
+      Inspired from : https://github.com/raywenderlich/SKTUtils/blob/master/SKTUtils/CGFloat%2BExtensions.swift
+     */
+    public static func shortestAngleBetweenInRadians(angle1InRadians: CGFloat, angle2InRadians: CGFloat) -> CGFloat {
```

Comment:
Answer to the second question : 

If CGFloats are treated as angles in the Core Graphics package(which our existing extensions like degreesToRadians actually do) , there should exist at least one method that computes the angular distance between the two.

Answer to the first question : -

The first option seems like a valid alternative.

So are we in a situation to reach consensus?